src/number_theory/frobenius_number.lean

```diff
@@ -6,6 +6,8 @@ Authors: Alex Zhao
 import data.nat.modeq
 import group_theory.submonoid.basic
 import group_theory.submonoid.membership
+import tactic.ring
+import tactic.zify
```

Comment:
I assume these imports are new here because some upstream imports got deleted?

Reply:
Was included in #16465.